apps/client/ui/InputSearch/InputSearch.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo } from 'react'
+import clsx from 'clsx'
+import {
+  BASE_INPUT_CLASSES,
+  InputProps,
+  getSizeClasses,
+} from '../InputText/InputText'
+import { SearchIcon } from '../Icons/Search/Search'
+
+export const InputSearch = ({
```

Comment:
in case of reusable components props it's good pattern to pass `...props` to root element of the component. then you will be able to interact with it at least in case you will need to pass some styles (sometimes we need to pass some margin in usage to component) 😄 
if you need to have also for `input` you may create `inputProps` prop and spread it to `input` 👌

Reply:
explanation needed 🚨